utils/utils/src/main/scala/pl/touk/nussknacker/engine/util/json/ToJsonEncoder.scala

```diff
@@ -23,73 +22,29 @@ case class ToJsonEncoder(
     highPriority: PartialFunction[Any, Json] = Map()
 ) {
 
-  private val safeString     = safeJson[String](fromString)
-  private val safeLong       = safeJson[Long](fromLong)
-  private val safeInt        = safeJson[Int](fromInt)
-  private val safeDouble     = safeJson[Double](fromDoubleOrNull)
-  private val safeFloat      = safeJson[Float](fromFloatOrNull)
-  private val safeBigDecimal = safeJson[java.math.BigDecimal](a => fromBigDecimal(a))
-  private val safeBigInt     = safeJson[java.math.BigInteger](a => fromBigInt(a))
-  private val safeNumber     = safeJson[Number](a => fromDoubleOrNull(a.doubleValue())) // is it correct?
-
   private val optionalCustomisations =
     ServiceLoader.load(classOf[ToJsonEncoderCustomisation], classLoader).asScala.map(_.encoder(this.encode))
 
+  private val additionalEncoders: PartialFunction[Any, Json] = { case value: DisplayJson => value.asJson }
+
   def encode(obj: Any): Json = optionalCustomisations
-    .foldLeft(highPriority)(_.orElse(_))
+    .foldLeft(highPriority.orElse(additionalEncoders))(_.orElse(_))
```

Comment:
Changed to use ToJsonEncoderWithFallback from components-api as much as possible.

This class `ToJsonEncoder` provides additional functionality - customization and preserves old interface.